native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -1166,7 +1163,7 @@ static apr_status_t proxy_cluster_try_pingpong(request_rec *r, proxy_worker *wor
     status = ap_proxy_connect_backend(scheme, backend, worker, r->server);
     /* Do nothing: 2.4.x has the ping_timeout and conn_timeout */
     if (status != APR_SUCCESS) {
-        ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, r->server, "proxy_cluster_try_pingpong: can't connect to backend");
+        ap_log_error(APLOG_MARK, APLOG_ERR, 0, r->server, "proxy_cluster_try_pingpong: can't connect to backend");
```

Comment:
Hm in case we can't connect to the back-end we are going to fill the log with a error message and the error is probably not a configuration issue...

Reply:
Ok, so maybe `APLOG_NOTICE`?

Reply:
I would keep the APLOG_DEBUG here, otherwise we will fill up the error_log with this message (you get one for each request being proxyied)